src/OpenSSL/crypto.py

```diff
@@ -1425,9 +1426,35 @@ def get_extension(self, index):
 X509Type = X509
 
 
+class X509StoreFlags(Enum):
+    """ Flags for X509 verification
```

Comment:
- `"""` please on single lines
- more explanation please?
- crosslink with set_flags (and backlink too)
- add something like “See the OpenSSL docs for the meaning.” and link some words to that URL